src/sdk/model/o365/o365-user.ts

```diff
@@ -57,6 +57,14 @@ export class O365User {
     return this._json.archived;
   }
 
+  /**
+   * Get whether this O365 user is an active user. False for active.
```

Comment:
wrong doc. should be "Get whether this O365 user is a licensed user in the Microsoft 365 organization it belongs. False for unlicensed."